go.mod

```diff
@@ -1,33 +1,98 @@
 module github.com/ramendr/ramen
 
-go 1.16
+go 1.17
 
 require (
 	github.com/aws/aws-sdk-go v1.38.41
 	github.com/backube/volsync v0.3.0
+	github.com/csi-addons/kubernetes-csi-addons v0.3.1-0.20220510101231-848e8b27d395
 	github.com/csi-addons/volume-replication-operator v0.1.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.3
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.15.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.19.0
 	github.com/open-cluster-management/api v0.0.0-20210527013639-a6845f2ebcb1
-	github.com/operator-framework/api v0.10.0
+	github.com/operator-framework/api v0.14.1-0.20220413143725-33310d6154f3
```

Comment:
This should be a non-commit version? It is possibly getting pulled from somewhere, any pointers?

Reply:
This is because, it is the version that csiaddons is pulling. See here https://github.com/csi-addons/kubernetes-csi-addons/blob/main/go.mod#L149